policyengine_us/variables/gov/usda/csfp/commodity_supplemental_food_program.py

```diff
@@ -0,0 +1,14 @@
+from policyengine_us.model_api import *
+
+
+class commodity_supplemental_food_program(Variable):
+    value_type = float
+    entity = Person
+    unit = USD
+    definition_period = YEAR
+    label = "Commodity Supplemental Food Program"
+    defined_for = "commodity_supplemental_food_program_eligible"
+
```

Comment:
This should work in place of the formula. Please remove the unit test and add a simple integration test to verify.
```suggestion
    adds = "gov.usda.csfp.amount"
```

Reply:
I have suggested this prior but it returned `TypeError: 'int' object is not iterable when I tried to use adds = "gov.usda.csfp.amount"`

Reply:
it should work now as https://github.com/PolicyEngine/policyengine-core/issues/102 is closed

Reply:
I ran `make install`, but it still does not work.

Reply:
my mistake - it's `adds = ["gov.usda.csfp.amount"]`